mavros_msgs/msg/HilGPS.msg

```diff
@@ -0,0 +1,17 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_GPS
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_GPS
+
+std_msgs/Header header
+uint8 fix_type
+sensor_msgs/NavSatFix fix
```

Comment:
HilGPS.msg already not sensor_msgs. Give a link to src where sensor_msgs included in other message is more handy.